### ISSUE TITLE
Questions regarding SQA3D reproduction results

### ISSUE THREAD

**Issue by Dai434** [LZ-CH/DSPNet#9]
During replication, I noticed slight discrepancies between my test results on the SQA3D dataset and those reported in the paper, particularly regarding the 'how' metric. 
In my setup, the batch size and number of workers were both set to 12, and the model was trained on a single RTX 4090 without loading the pretrained VoteNet weights.
<img width="626" height="106" alt="Image" src="https://github.com/user-attachments/assets/23111482-1dac-44a7-83e2-f1aa2d2b6c32" />
